module/modmanager/manager.go

```diff
@@ -780,10 +780,18 @@ func (m *module) startProcess(
 		return err
 	}
 
+	// We evaluate the Module's ExePath absolutely in the viam-server process so that
+	// setting the CWD does not cause issues with relative process names
+	absoluteExePath, err := filepath.Abs(m.cfg.ExePath)
```

Comment:
If there is a cleaner way to fix this, happy to switch to that.

This fixes:
`m.cfg.ExePath = "module/exe"` (relative path)
`pconf.Name = "module/exe"` (still relative) (used to work)
`pconf.CWD = "module"` (will cause problems because now we will try to find the relative path `module/exe` from within `module` )

Reply:
Maybe just always set CWD to the VIAM_MODULE_ROOT... e.g. the path were the module is extracted. Then, TRY to interpret the path in a relative way for the exe. If that isn't actually found, then try other ways to "interpret" the path for backwards compatibility.

Thoughts?

Reply:
But then it has the same problem with registry modules and local modules having different working directories :[ So then it will work locally and then only fail after it has been uploaded to the registry.

Reply:
I think a consistent experience is the most important thing